core/domain/platform_parameter_domain.py

```diff
@@ -451,21 +526,25 @@ def _is_first_flavor_smaller(self, flavor_a, flavor_b):
         Returns:
             bool. True if the first flavor is smaller.
         """
-        return (
+        return (bool(
             ALLOWED_APP_VERSION_FLAVORS.index(flavor_a) <
-            ALLOWED_APP_VERSION_FLAVORS.index(flavor_b)
-        )
+            ALLOWED_APP_VERSION_FLAVORS.index(flavor_b))
+            )
 
```

Comment:
```python
        flavor_a_index = ALLOWED_APP_VERSION_FLAVORS.index(flavor_a)
        flavor_b_index = ALLOWED_APP_VERSION_FLAVORS.index(flavor_b)
        return bool(flavor_a_index < flavor_b_index )
```

Also is the `bool()` really needed?

Reply:
yep if we don't put the bool (...) it doesn't work

Reply:
done